boards/arm/qemu_cortex_r5/board.cmake

```diff
@@ -9,6 +9,7 @@ set(QEMU_CPU_TYPE_${ARCH} cortex-r5)
 set(QEMU_FLAGS_${ARCH}
   -nographic
   -machine arm-generic-fdt
+  -icount shift=3,align=off,sleep=off -rtc clock=vm
```

Comment:
No complaints here, though IIRC there are still some issues in the main -icount patch.  Does this work for everything?

Reply:
This change is actually from #22904.

There is an issue with the WFI instruction in the Xilinx QEMU when icount mode is used, but this only affects one Zephyr test (tests/kernel/context) and does not really have a real impact in terms of functionality.

https://github.com/zephyrproject-rtos/zephyr/pull/22904#issuecomment-591209160